inst/examples/02-components.Rmd

```diff
@@ -850,3 +850,15 @@ knitr::include_app('https://yihui.shinyapps.io/miniUI/', height = '600px')
 Again, you will see a live app if you are reading an HTML version of this book, and a static screenshot if you are reading other types of formats. The above Shiny app was created using the **miniUI** package [@R-miniUI], which provides layout functions that are particularly nice for Shiny apps on small screens. If you use normal Shiny layout functions, you are likely to see vertical and/or horizontal scrollbars in the iframes because the page size is too big to fit in an iframe. When the default width of the iframe is too small, you may use the chunk option `out.width` to change it. For the height of the iframe, use the `height` argument of `include_url()`/`include_app()`.
 
 Shiny apps may take even longer to load than usual URLs. You may want to use a conservative value for the `delay` option, e.g., 10. Needless to say, `include_url()` and `include_app()` require a working Internet connection, unless you have previously cached the chunk (but web pages inside iframes still will not work without an Internet connection).
+
+## HTML 404 pages {#html-404}
+
+By default, both the `gitbook()` (Section \@ref(gitbook-style)) and `bs4_book()` (Section \@ref(bs4-book)) HTML output formats include a 404 page for readers who try to access a page in your book that cannot be found. The default 404.html is created using simple content (a header, and a body of 2 paragraphs). 
```

Comment:
```suggestion
If a readers try to try to access a page in your book that cannot be found (e.g a link to a page that no more exists), a browser will display a [404 error](https://en.wikipedia.org/wiki/HTTP_404) as it can’t find the requested web page. This 404 error is displayed on a 404 page that each browser has a default for. You may want to customized this page to be in the style of your HTML book. 
Most web serving platforms (e.g. Netlify, Github Pages, or Gitlab Pages) will use a file named `404.html` in the root of your website as a custom error page if you provide it. 

With HTML book format, **bookdown** creates a default `404.html`  using simple content (a header, and a body of 2 paragraphs).  
```